script/dev_deploy_conf_example.yml

```diff
@@ -0,0 +1,61 @@
+hosts:
+  remote_ssh_host1: 192.168.10.1
+  remote_ssh_host2: 192.168.10.2
+  remote_ssh_host3: 192.168.10.3
+  remote_ssh_host4: 192.168.10.4
+zookeeper:
+  persistent-dir: "/data/zookeeper"
+  hosts:
+    - remote_ssh_host2
+    - remote_ssh_host3
+    - remote_ssh_host4
+  enable-metrics-provider: true
+hstore:
+  image: hstreamdb/hstream:v0.7.1
```

Comment:
please update hstream version to v0.8.0, also update the `example.json`

Reply:
fixed